lib/matplotlib/legend.pyi

```diff
@@ -19,7 +19,7 @@ from matplotlib.transforms import (
 import pathlib
 from collections.abc import Iterable
 from typing import Any, Literal, overload
-from .typing import ColorType
+from .typing import ColorType, LegendLocType
```

Comment:
This line is technically first-party and should be grouped with the others (though the rest of these of this block are stdlib and belong first, also.)

Reply:
done